### DISCUSSION TITLE
Need urgent help : server migration issues 😭

### DISCUSSION THREAD

**Discussion by Pawan113** [chamilo/chamilo-lms#5336] [General]
So, I started migration of my LMS to another hosting plan.

It took me 3 days to do it correctly or so I thought. Now I can see correct front page, but can't enter it using my admin credentials.

Please help, what to do?!!

Domain : oracleias.in

**Comment by Pawan113** [comment]
Now it doesn't even show front page. Fatal error: Declaration of Doctrine\DBAL\Driver\PDOConnection::query() must be compatible with PDO::query(string $query, ?int $fetchMode = null, mixed ...$fetchModeArgs): PDOStatement|false in /home2/oracl1qc/public_html/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php on line 70

Is all shown. Hosting company is not helping. 

I was stupid to migrate this. How do I make this work again? If there is a professional, I would be ok to pay for their efforts.

**Comment by NicoDucou** [comment]
Hi @Pawan113 

You have a list of official provider here : https://chamilo.org/en/providers/

Best regards
